content/en/blogs/aws-kubernetes.md

```diff
@@ -0,0 +1,251 @@
+---
+title: 'Using KubeKey to Deploy an HA Kubernetes Cluster on AWS'
```

Comment:
According to the SEO recommendations, I propose to use this title: `How to Deploy an HA Kubernetes Cluster on AWS with KubeKey`